code/index.html

```diff
@@ -1,30 +1,126 @@
-<!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <link rel="stylesheet" href="Normalize.css/normalize.css">
+    <link rel="stylesheet" href="./style.css">
+    
+</head>
+
+<body>
+<!-- the text on the image -->
+    <section class="hero" id="sec-1">
+        <h1 class="bigtext">Need a smooth shave?</h1>
+
+<!-- the scrool and linked it to section to which is the forms -->
+        <a href="#sec-2">
+            <div class="scroll-text-box">
+            </div>
+            <div class="scroll-down"> </div>
+        </a>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+<!-- second section with the forms/radio-buttons/checkboxes/selector and subscribe button -->
+    <section class="subscribe" id="sec-2">
+        <div>
+            <div class="bg">
+                <form action="http://httpbin.org/anything" method="post" class="forms">
```

Comment:
The form is has a very nice layout, you used several different input types and aligned them nicely. I See that you skipped labels next to your text-inputs. which I personally think looks great but a question that occured to me was if that affects the semantic reading of it so for example screen readers? But maybe they do! 🤔